src/main/webapp/app/lecture/lecture-update.component.html

```diff
@@ -1,131 +1,132 @@
-                                <fa-icon [icon]="faHandShakeAngle" />&nbsp;<span jhiTranslate="artemisApp.lecture.home.switchToGuidedModeLabel"></span>
-                            </button>
-                        </div>
+        <form name="editForm" role="form" (ngSubmit)="save()" #editForm="ngForm">
+            <div class="d-flex">
+                <div class="flex-grow-1">
+                    <div class="d-flex align-items-center">
+                        <h1 id="jhi-lecture-heading" [jhiTranslate]="isEditMode() ? 'artemisApp.lecture.home.editLabel' : 'artemisApp.lecture.home.createLabel'"></h1>
+                        <jhi-documentation-button [type]="documentationType" />
                     </div>
                 </div>
+            </div>
+            <jhi-form-status-bar [formStatusSections]="formStatusSections" />
+            <div>
                 <div>
-                    <h2 id="artemisApp.lecture.wizardMode.steps.titleStepTitle"><span jhiTranslate="artemisApp.lecture.wizardMode.steps.titleStepTitle"></span></h2>
-                    <p><span jhiTranslate="artemisApp.lecture.wizardMode.steps.titleStepMessage"></span></p>
-                    <jhi-lecture-title-channel-name [lecture]="lecture()" />
-                    <div class="form-group" id="field_description">
-                        <label class="form-control-label" jhiTranslate="artemisApp.lecture.description" for="field_description"></label>
-                        <jhi-markdown-editor-monaco
-                            class="markdown-editor"
-                            [domainActions]="domainActionsDescription"
-                            [(markdown)]="lecture().description"
-                            (markdownChange)="updateIsChangesMadeToTitleOrPeriodSection()"
-                        />
-                    </div>
-                    <jhi-lecture-update-period [lecture]="lecture()" [validateDatesFunction]="onDatesValuesChanged" />
+                    <h2 id="artemisApp.lecture.sections.title"><span jhiTranslate="artemisApp.lecture.sections.title"></span></h2>
+                    <p><span jhiTranslate="artemisApp.lecture.sections.titleDescription"></span></p>
                     @if (lecture().course) {
                         <div class="form-group">
                             <label class="form-control-label" jhiTranslate="artemisApp.lecture.course" for="field_course"></label>
-                            <div class="d-flex">
-                                <input id="field_course" disabled type="text" class="form-control" name="course" [(ngModel)]="lecture().course!.title" />
-                            </div>
+                            @if (lecture().course?.title) {
+                                <div class="d-flex">
+                                    <input id="field_course" disabled type="text" class="form-control" name="course" [(ngModel)]="lecture().course!.title!" />
```

Comment:
Please avoid the non-null assertion operator (`!`)

Reply:
I do unfortunately now know how to get rid of the non-null assertion.

I have even tried it with `lecture().course && lecture().course.title`, but the client will not be built
```
✘ [ERROR] NG2: Object is possibly 'undefined'. [plugin angular-compiler]

    src/main/webapp/app/lecture/lecture-update.component.html:22:147:
      22 │ ...orm-control" name="course" [(ngModel)]="lecture().course.title" />
```

How can it be solved without non null assertion?

Reply:
I figure the problem is the two-way binding by `[(ngModel)]` and the nested signal with the optional properties course and title. You could do some additional research, but I would suggest to create a signal or `model()` input for the course title itself and synchronize `courseTitle` with `lecture().course.title` when lecture changes, and another to update `lecture().course.title` when `courseTitle` changes. Bind `courseTitle` to the input field using `[(ngModel)]=courseTitle`.  

Let me know if this works.

Reply:
Thanks for the suggestion, I adjusted it according to the suggestion :)